From 4ff99ed33f01580dcc52c68e3821f8fdf97f3bbd Mon Sep 17 00:00:00 2001
From: Xiaolei Wang <xiaolei.wang@windriver.com>
Date: Fri, 3 Apr 2020 18:59:13 +0800
Subject: [PATCH] drm/imx/dpu: fix warning: 'm' may be used uninitialized in
 this function in dpu-kms.c

Initialize m, otherwise the compiler issues warning: 'm' may be used
uninitialized in this function when it detects that it is assigned to a
variable

Signed-off-by: Xiaolei Wang <xiaolei.wang@windriver.com>
---
 drivers/gpu/drm/imx/dpu/dpu-kms.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/imx/dpu/dpu-kms.c b/drivers/gpu/drm/imx/dpu/dpu-kms.c
index 1709fed6308e..9598cfa08c07 100644
--- a/drivers/gpu/drm/imx/dpu/dpu-kms.c
+++ b/drivers/gpu/drm/imx/dpu/dpu-kms.c
@@ -208,7 +208,7 @@ dpu_atomic_assign_plane_source_per_crtc(struct drm_plane_state **states,
 	dpu_block_id_t blend;
 	unsigned int sid, src_sid;
 	unsigned int num_planes;
-	int i, j, k, l, m;
+	int i, j, k, l, m = 0;
 	int total_asrc_num;
 	int s0_layer_cnt = 0, s1_layer_cnt = 0;
 	int s0_n = 0, s1_n = 0;
-- 
2.17.1

